<launch>
  <node pkg="corner_detect" type="lasescan" name="laser_scan" output="screen"/>
</launch>
